lib/magma/commands.rb

```diff
@@ -127,4 +127,26 @@ def setup(config)
       Magma.instance.load_models
     end
   end
+
+  class Unload < Etna::Command
+    usage 'Run Unloader to dump the dataset of a model into a gzipped tsv.
+      *args - arg[0]: project_name, arg[1]: model_name, args[2]: gzip?, arg[3]: file'
+
+    def execute(*args)
+      begin
```

Comment:
this could be execute(project_name, model_name, gzip?, directory), so you don't have to use args[0] etc. below